Prisma 6.16.0 types issue with prisma-client generator

With the move to 6.16.0 the files in the generated Prisma Client `models` folder now use `import type * as runtime from "@prisma/client/runtime/library"` instead of `import * as runtime from "@prisma/client/runtime/library"`. In the modified Zenstack version of these files we prepend `import $Types = runtime.Types` (unclear why this is needed). This now raises the following type error: 

`error TS1380: An import alias cannot reference a declaration that was imported using 'import type'.`

I'd make a fix for this but I'm unclear if this import declaration is really needed or what purpose it serves. 

**Environment:**

-   ZenStack version: 2.18.1
-   Prisma version: 6.16.0
-   Database type: Postgresql

--- a/packages/schema/src/plugins/enhancer/enhance/index.ts
+++ b/packages/schema/src/plugins/enhancer/enhance/index.ts
@@ -612,7 +612,7 @@ export type Enhanced<Client> =
                 throw new PluginError(name, `Unexpected syntax list structure in ${fileName}`);
             }
 
-            const statements: (string | StatementStructures)[] = ['import $Types = runtime.Types;'];
+            const statements: (string | StatementStructures)[] = [];
 
             // Add import for json-types if this model has JSON type fields
             const modelWithJsonFields = this.modelsWithJsonTypeFields.find((m) => m.name === d.name);
@@ -1032,7 +1032,7 @@ export type Enhanced<Client> =
             // Check if the field in the source is optional (has a `?`)
             const isOptionalInSource = new RegExp(`(${field.name}\\?\\s*):`).test(source);
             if (isOptionalInSource) {
-                replaceValue += ' | $Types.Skip';
+                replaceValue += ' | runtime.Types.Skip';
             }
 
             return source.replace(new RegExp(`(${field.name}\\??\\s*):[^\\n]+`), replaceValue);

